tools/autograd/gen_autograd.py

```diff
@@ -68,7 +70,8 @@
 # this list contains both the root view functions and any that are purely composed
 # of viewing functions, and is used by the JIT to determine when an operator
 # returns a view of its inputs
```

Comment:
Could you add a note in this comment that this link contains all the functions that **can** return a view they might not return views in some cases (like contiguous).

Reply:
Good point.